src/types.js

```diff
@@ -168,9 +171,20 @@ function FuncValue(source) {
         },
         show: function () {
             if (typeof(source) === "function") {
-                return "[<internal>]";
+                return "[" + name + "]";
```

Comment:
Isn't that a little misleading? That makes it seems like it's a wrapped version. Just `return name;` is surely better.